vuepress/docs/next/docs/getting-started/entando-cli.md

```diff
@@ -51,18 +50,17 @@ ent [command] [subcommand] [name] [flags]
 > ent Commands:
   - app                  => Helper for managing an Entando App
   - app-info             => Displays information about an Entando App
-  - bundle               => Helper for managing bundle functions. See below for more details
+  - bundle               => Helper module for managing bundle functions. (See below for more details)
+  - bundler              => export of resources from a running instance for v1 bundles only
```

Comment:
@nshaw only mentioned version here for bundler. Thinking I'll get into the details of the v1/v5 issues in the project management page. Thought I'd keep it more generic here. Any thoughts?

Reply:
@jyunmitch Maybe even there we can use the phrase "git-based bundles (v1)". I think git-based versus docker-based is going to be the most easily understood.

Reply:
Maybe - "export resources from a running instance as a git-based (v1) bundle." Then the previous line can be "Helper module for managing docker-based (v5) bundles. (See below for more details)"